Update ithim_setup_parameters.R

jracp opened the pull request:
- Adding error_handling.R function
- Checking if the default values of PM_CONC_BASE and PM_TRANS_SHARE are changed; otherwise, through a warning message
- Adding a new variable called variables containing names of inputs in string
- Adding a for loop to replace nine if-commands

robj411 commented:
Hi,
Please check parameter names e.g. lines 15, 26, 33; 33, 39; 31, 56, and distributions, e.g. lines 48, 55, 58.
Thanks,
Rob

jracp commented:
- Fixed the variable names
- Changed _variables_ to _normVariables_ and _betaVariables_ to handle normal and beta distributions, respectively
- Handled both distributions in the same programming fashion
